src/pages/dex/_components/PoolPairsTable.tsx

```diff
@@ -130,24 +138,19 @@ function PoolPairRow ({ data }: { data: PoolPairData }): JSX.Element {
         />
       </OverflowTable.Cell>
       <OverflowTable.Cell className='align-middle text-right'>
-        {(() => {
-          const tokenPrice = getTokenPrice(data.tokenA.symbol, '1').toString()
-          return (
-            <NumberFormat
-              value={tokenPrice}
-              displayType='text'
-              thousandSeparator
-              fixedDecimalScale
-              decimalScale={Number(tokenPrice) > 100 ? 0 : 2}
-              prefix='$'
-            />
-          )
-        })()}
+        <NumberFormat
+          value={tokenPrice}
+          displayType='text'
+          thousandSeparator
+          fixedDecimalScale
+          decimalScale={tokenPrice > 100 ? 0 : 2}
+          prefix='$'
+        />
```

Comment:
To counter the rounding vs cutting issue - I suggest we can try modifying to this, instead of letting NumberFormat cut off the decimals without rounding up
```suggestion
  <NumberFormat
          value={tokenPrice.isGreaterThan(100) ? tokenPrice.toFixed(0,BigNumber.ROUND_HALF_UP) : tokenPrice.toFixed(2, BigNumber.ROUND_HALF_UP)}
          displayType='text'
          thousandSeparator
          prefix='$'
        />
```

Reply:
Yep i created an issue for this https://github.com/DeFiCh/scan/issues/1082.

Reply:
Ok pushed this to display the correct rounding number in the DEX table/card.